scratch/xbgpu/benchmarks/beamform_bench.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python3
+
+import argparse
+import itertools
+
+import katsdpsigproc.accel
+
+from katgpucbf.xbgpu.beamform import BeamformTemplate
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--array-size", type=int, default=80, help="Antennas in the array [%(default)s]")
+    parser.add_argument(
+        "--channels-per-substream", type=int, default=16, help="Channels processed by one engine [%(default)s]"
```

Comment:
I know this is your first pass (ha!) at this benchmark script, worth asking why this value is so low?

Reply:
If I understand correctly, 1024 channels is the standard mode for pulsar timing, and with 80 antennas we expect to need 64 XB-engines, which gives 16 channels per engine.